packages/css-variables/README.md

```diff
@@ -0,0 +1,25 @@
+# @theme-ui/css-variables
```

Comment:
I think the package name could be `@theme-ui/custom-properties` to be a little more explicit about what it's intended for